.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
 script: make travis
```

Comment:
In theory doing make should work on nix if the proper shebang is present,
shouldn't it?
On Mar 5, 2015 12:36 PM, "Joshua Mervine" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/MaxCDN/bootstrap-cdn/pull/509#discussion_r25853968>:
>
> > @@ -1,4 +1,7 @@
> >  language: node_js
> >  node_js:
> >    - "0.10"
> > +  - "0.12"
> >  script: make travis
>
> I understand your view on that. I hesitate because in 9000 other things I
> do, it's "make ..." so it's basically muscle memory for me. For my own
> sanity I try to keep things consistent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MaxCDN/bootstrap-cdn/pull/509/files#r25853968>.
>